packages/commonwealth/client/scripts/views/components/quill/render_quill_delta.ts

```diff
@@ -106,7 +108,12 @@ export const renderQuillDelta = (
                       target: openLinksInNewTab ? '_blank' : '',
                       noreferrer: 'noreferrer',
                       noopener: 'noopener',
-                      onclick: (e) => {
+                      onclick: async (e) => {
+                        console.log('click');
```

Comment:
`console.log('reviewed for the uber eats');`

Reply:
I believe that is a comment, not a review.